pkg/descheduler/strategies/node_affinity.go

```diff
@@ -74,6 +74,7 @@ func RemovePodsViolatingNodeAffinity(ctx context.Context, client clientset.Inter
 
 		switch nodeAffinity {
 		case "requiredDuringSchedulingIgnoredDuringExecution":
```

Comment:
lgtm actually, but i just wonder why this name `requiredDuringSchedulingIgnoredDuringExecution` so long...

Reply:
Chosen by the designers: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity

Reply:
get it. look good to me now~
/lgtm